test/test_cli.py

```diff
@@ -0,0 +1,140 @@
+from os.path import realpath
+from pathlib import Path
+from unittest import TestCase
+import biggerquery as bgq
+from biggerquery.cli import *
+
+TEST_MODULE_SRC = 'test_module'
+
+
+class CacheBuilderTestCase(TestCase):
+
+    def test_walk_module_files(self):
+        # given
+        path = Path(TEST_MODULE_SRC)
+
+        # when
+        res = walk_module_files(path)
+
+        # then
+        res_as_list = list(res)
+        absolute = str(Path(TEST_MODULE_SRC).absolute())
+        expected = [(absolute, "__init__.py"), (absolute, "Unused1.py"), (absolute, "Unused2.py"),
+                    (absolute, "Unused3.py")]
+        self.assertEqual(expected, res_as_list)
+
+        for (path, name) in res_as_list:
+            self.assertEqual('/', path[0], "Path should be absolute and start with /")
+            expected_ending = 'biggerquery/test/test_module'
+            self.assertEqual(expected_ending, path[-len(expected_ending):])
+
+    def test_walk_module_paths(self):
+        # given
+        path = Path(TEST_MODULE_SRC)
+
+        # when
+        res = walk_module_paths(path)
+
+        # then
+        res_as_list = list(res)
+        expected = ['test_module', 'test_module.Unused1', 'test_module.Unused2', 'test_module.Unused3']
+        self.assertEqual(expected, list(res_as_list))
+
+    def test_walk_modules(self):
+        # given
+        path = Path(TEST_MODULE_SRC)
+
+        # when
+        res = walk_modules(path)
+
+        # then
+        res = list(res)
```

Comment:
In case of a longer, non-obvious # then section, I suggest adding some comments that explain what is going on or wrapping assertions with meaningful function